my-app/src/components/game.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Board from './board.js';
-import calculateWinner from './utils.js';
+import calculateWinner from '/Users/hmyra/Documents/GitHub/innovance-frontend-internship/my-app/src/utils.js';
```

Comment:
Why did you add fullpath? Now this code won't be build other than your local environment. It should be relative path